### ISSUE TITLE
[QST] dropping rows?

### ISSUE THREAD

**Issue by ethem-kinginthenorth** [rapidsai/cudf#6779]
**What is your question?**
I am trying to drop rows and looking into this [documentation](https://docs.rapids.ai/api/cudf/nightly/api.html#cudf.core.dataframe.DataFrame.drop) and when I try `df.drop([0, 1]) , it errors out `NameError: column 0 does not exist`.  How do I drop rows? Code snippet:
```
import cudf
df = cudf.DataFrame({"A": [1, 2, 3, 4],
                     "B": [5, 6, 7, 8],
                     "C": [10, 11, 12, 13],
                     "D": [20, 30, 40, 50]})
df.drop([0,1])
```

**Comment by kkraus14** [comment]
@ethem-kinginthenorth what version of `cudf` do you have installed?

**Comment by ethem-kinginthenorth** [comment]
'0+untagged.1.gfa8e9fb'

**Comment by ethem-kinginthenorth** [comment]
I am using this container: `nvcr.io/nvidia/nvtabular:0.2`

**Comment by kkraus14** [comment]
@ethem-kinginthenorth can you run:
```
import cudf
print(cudf.__version__)
```
and share the output?

This was added in cudf 0.16: https://github.com/rapidsai/cudf/pull/6202

**Comment by kkraus14** [comment]
cc @EvenOldridge for nvtabular

**Comment by ethem-kinginthenorth** [comment]
![cudf_version](https://user-images.githubusercontent.com/44275619/99440752-01ef1680-28e5-11eb-9b24-c32c820b8467.PNG) When I do `pip list` I am seeing `0+untagged.1.gfa8e9fb`. am I missing something here? I was expecting something with just numbers (e.g., 0.0.4).

**Comment by kkraus14** [comment]
I would expect that as well. Will wait from someone from the nvtabular team to chime in here.

**Comment by EvenOldridge** [comment]
Our 0.2 container uses cuDF 0.15.  We'll have a new container out next week for our 0.3 release that uses 0.16 and will likely be cutting a 0.3.1 release to align with 0.17.

Not sure why the version doesn't show up correctly, although we do build cuDF ourselves from scratch so there may be an issue there that we should resolve.  @jperez999

**Comment by kkraus14** [comment]
Regardless, I'm going to close this issue as it's resolved as of cudf 0.16 which has been released for a while and 0.17 will be released in the next few weeks.
